src/chain.h

```diff
@@ -259,7 +259,7 @@ class CBlockIndex
     std::unordered_map<GroupElement, std::pair<uint256, std::vector<unsigned char>>> sparkTxHashContext;
 
     //! Values of coin serials spent in this block
-    sigma::spend_info_container sigmaSpentSerials;
+    std::unordered_map<Scalar, sigma::CSpendCoinInfo> sigmaSpentSerials;
```

Comment:
same here, we could leave `sigma::CScalarHash` in the code. It's not immediately clear if this mismatch could create a problem, but the removal is simply not worth it